### PR TITLE
Refactor framework symbols.

### PR DESCRIPTION
**Pull request by H1Gdev**
#### What does this implement/fix? Explain your changes.

Refactor framework symbols.

- Use [Preprocessor symbols](https://learn.microsoft.com/en-us/dotnet/standard/frameworks#preprocessor-symbols)

#### Any other comments?

Remove symbols defined by default.
https://learn.microsoft.com/dotnet/api/system.diagnostics.trace#remarks

### CODE REVIEW

File: Tests/Svg.UnitTests/Svg.UnitTests.csproj [hunk 1]
```diff
@@ -11,37 +11,6 @@
...
-    <DefineConstants>TRACE;DEBUG;NETFULL;NET452</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net452|AnyCPU'">
-    <DefineConstants>TRACE;RELEASE;NETFULL;NET452</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net462|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;NETFULL;NET462</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net462|AnyCPU'">
-    <DefineConstants>TRACE;RELEASE;NETFULL;NET462</DefineConstants>
```
Comment (mrbean-bremen) [inline Tests/Svg.UnitTests/Svg.UnitTests.csproj:24]: .NET 4.6.2 is supported until 2027, so I would leave these in.

**Review by mrbean-bremen — COMMENTED** [review]
Can you also change the CI tests from .NET 5 to .NET 7, and add a note to the release noters that the older frameworks are no longer supported?

**Comment by H1Gdev** [comment]
This is PR before upgrading from .NET 5.

File: Tests/Svg.UnitTests/Svg.UnitTests.csproj (hunk 1 above)
Comment (H1Gdev) [reply Tests/Svg.UnitTests/Svg.UnitTests.csproj:24]: .NET Framework 4.6.2 is still valid.
https://github.com/svg-net/SVG/blob/71ffa1148a29eb1aab29ec09a09a98de2b5b7ddd/Tests/Svg.UnitTests/Svg.UnitTests.csproj#L4

I remove definition of symbols defined in vs project, or unused.

Comment (mrbean-bremen) [reply Tests/Svg.UnitTests/Svg.UnitTests.csproj:24]: True, you are right.

**Comment by mrbean-bremen** [comment]
> This is PR before upgrading from .NET 5.

So you want to make a new PR for that?

**Comment by H1Gdev** [comment]
> So you want to make a new PR for that?

Yes.
I've tested upgrade from .NET5 to .NET6 and know it's not just `<TargetFrameworks>` changes.
So I do not include it in this PR.

**Review by mrbean-bremen — APPROVED** [review]

**Pull request merged**